src/core/core.scale.autoskip.js

```diff
@@ -16,7 +16,8 @@ import {_factorize} from '../helpers/helpers.math';
  */
 export function autoSkip(scale, ticks) {
   const tickOpts = scale.options.ticks;
-  const ticksLimit = tickOpts.maxTicksLimit || determineMaxTicks(scale);
+  const detirmendMaxTicks = determineMaxTicks(scale);
```

Comment:
Does not sound a valid word to me, did you mean determined? (I could be wrong too)

Reply:
Yeah, you are right, updated it 👍🏻